nautobot/extras/filters.py

```diff
@@ -327,6 +327,15 @@ def __init__(self, custom_field, *args, **kwargs):
             # Contains handles lists within the JSON data for multi select fields
             self.lookup_expr = "contains"
 
+    def filter(self, qs, value):
+        NULL_FIELDS = ("null", "0")
+
+        if "_custom_field_data" in self.field_name and value in NULL_FIELDS:
+            lookup_exact = "%s__%s" % (self.field_name, "exact")
+            lookup_isnull = "%s__%s" % (self.field_name, "isnull")
+            return self.get_method(qs)(Q(**{lookup_exact: None}) | Q(**{lookup_isnull: True}))
```

Comment:
This seems like it's more complicated than needs be. Why not just:

```suggestion
            return self.get_method(qs)(Q(**{f"{self.field_name}__exact": None}) | Q(**{f"{self.field_name}__isnull": True}))
```